### ISSUE TITLE
Plugin not working without timezone suffix

### ISSUE THREAD

**Issue by cmeadows** [mattermost/mattermost-plugin-walltime#12]
I have v0.1.1 installed on Mattermost Server 5.8. This does not seem to work without a timezone as the suffix. Since there are no settings, I assume there is a bug somewhere?

For example, your examples on the README do not change the dates on our server. The following does not work.
```
Lets meet tomorrow at noon
```
But the following does work 
```
Lets meet tomorrow at noon edt
```

**Comment by mastercoms** [comment]
This issue is also happening for me.

**Comment by mkraft** [comment]
@mastercoms This is actually a requirement in the current implementation of this plugin because there is currently no way other reference for determining the time that the author intended otherwise. Think of it this way:

I say "meet me tomorrow at noon" and my current timezone in Mattermost is set to UTC+1 because I'm visiting London in the summer. The plugin would say "to the author his local day was May 29th and to him noon meant noon in UTC+1". Then it converts it for you to your walltime.

Then I fly home to Toronto and my timezone in Mattermost changes to UTC-4. Suddenly the reference for the plugin is "to the author his local day was May 29th and to him noon meant noon in UTC-4". And the time you are shown changes.

Put simply until Mattermost stores the currently configured timezone of the author for each post in the database we need some way to know the timezone that the author intended.

Closing this as it's not a bug. Thank you.
